app/viewmodels/ResultsPageSummary.scala

```diff
@@ -32,6 +32,10 @@ class ResultsPageSummary(from: LocalDate, to: LocalDate, day: Boolean = true)
     cashTransactionsResultRow(CashTransactionDates(from, to), fullStop, day)
   }
 
+  def rowsV2(fullStop: Boolean = true): SummaryListRow = {
```

Comment:
please rename the parameter as isFullStop and in other related private method as well. Boolean parameter should be following these naming convention to tell the user regarding type of parameter